tests/storage/hotplug.go

```diff
@@ -1280,7 +1282,8 @@ var _ = Describe(SIG("Hotplug", func() {
 					libdv.WithRegistryURLSource(url),
 					libdv.WithStorage(
 						libdv.StorageWithStorageClass(storageClass),
-						libdv.StorageWithVolumeSize("256Mi"),
+						libdv.StorageWithVolumeSize("500Mi"),
+						libdv.StorageWithVolumeMode(k8sv1.PersistentVolumeFilesystem),
```

Comment:
Same question about this?

Reply:
Yeah I'd say we do, better to be more imperative with the volume mode here since we want FS for this DV.

Reply:
We do? I think @awels intention is to use whatever is available in the test config (block/fs)

Reply:
We need to rename the disk image from this DV, so at least based on how the test is structured, it needs to be FS to access it and rename it. That's why we use the storage class from GetRWOFileSystemStorageClass, assuming the default volume mode will be FS.

Reply:
Was looking at the wrong test :man_facepalming: